### DISCUSSION TITLE
Custom Supports Pass Through Model

### DISCUSSION THREAD

**Discussion by amacey** [5axes/CustomSupportCylinder#6] [Ideas]
These custom supports are a game-changer for Cura - thanks!

I noticed a possible issue when placing them though. They seem to pass through the model and always touch the buildplate, even if you don't want the sections of the model below to actually be supported. As a workaround, I tried adding a support blocker in the lower section of the model, but these custom supports seem to ignore the blocker. 

Is this something that could be addressed in a future version?

Thanks!

**Comment by 5axes** [comment]
The plugin has no influence on the support creation. This is manage by Cura. The goal of the plugin is to creat a "custom support" geometry nothing more. So this request must be adressed to the Cura developpement team : [https://github.com/Ultimaker/Cura/issues](https://github.com/Ultimaker/Cura/issues)
